src/tests/ftest/control/dmg_telemetry_io_basic.yaml

```diff
@@ -0,0 +1,41 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-A
+timeout: 150
+server_config:
+  name: daos_server
+  servers:
+    targets: 1
```

Comment:
Any specific reason we are using 1 target here? I think on VMs max we can use is 2 targets per VM so may be nice to make use of max available resources.

Reply:
updated.